solutions/fully-configurable/data.tf

```diff
@@ -1,3 +1,9 @@
+locals {
```

Comment:
can we move the code in data.tf into main.tf - we usually keep all core code in that file